clang/examples/DPCT/Driver/cuMemcpy2D.cu

```diff
@@ -1,5 +0,0 @@
-void test(const CUDA_MEMCPY2D *pm) {
```

Comment:
while remove 5 sample files. (4 in Driver folder and 1 in Runtime folder)

Reply:
Now the CUDA_MEMCPY2D cannot be migrated correctly.